### DISCUSSION TITLE
binary units in all related progress columns

### DISCUSSION THREAD

**Discussion by Poshi** [Textualize/rich#2034] [Ideas]
I realized that you can ask `rich.progress.DownloadColumn` to interpret data in binary units (base 2) or decimal units (base 10). But I cannot see anything similar for `TransferSpeedColumn`, `FileSizeColumn` and `TotalFileSizeColumn`.
Don't you think that these four columns should share the same ability?

**Comment by sbraz** [comment]
Hello, I've opened a feature request for this at https://github.com/Textualize/rich/issues/3466.
